setup.py

```diff
@@ -35,6 +35,7 @@
         "types-PyYAML",
         "sqlalchemy2-stubs",
         "banal >= 1.0.6, < 1.1.0",
+        "rigour >= 0.3.0, < 1.0.0",
```

Comment:
Maybe that’s a dumb question, but what’s the advantage of extracting this functionality into a separate package compared to a separate module in `followthemoney`? As it contains validation/normalization logic for FtM data, I guess most projects that would make use of this (inlcuding Aleph, nomenklatura, …) would require `followthemoney` anyways?

Reply:
That's a fair point, it could live in `followthemoney`. This is more meant to address a social issue than a technical one: at present, I'm treating FtM releases as expensive because they involve "y'all" - so I try to only push out a release here if it is super relevant.

The problem I have is that this leads to a lot of "plaque" in our upstream libs - for example when it comes to extra name and identifier processing code like here: https://github.com/opensanctions/nomenklatura/blob/main/nomenklatura/util.py#L95-L206 

The idea with `rigour` is to combine all the intel about the types and how to process them into one place, and to basically make a promise that the upstream compatibility with FtM will be preserved, while not putting a limitation on the amount of e.g. name processing stuff that we can add as needed.

Reply:
Thanks for clarifying, that makes sense!